load_test.py

```diff
@@ -116,13 +116,15 @@ def main(args):
     """Load test CLI entrypoint."""
     args = utils.parse_args(args)
 
+    mp_mgr = mp.Manager()
```

Comment:
I think if we are using the manager class instead of context-based multi-processing, we should also use it to spin new processes (line #152). Have you tried using `mp_mgr` for the `create_procs`?